pallets/msa/src/lib.rs

```diff
@@ -161,6 +178,12 @@ pub mod pallet {
 		/// calculated value.
 		#[pallet::constant]
 		type MaxSignaturesStored: Get<Option<u32>>;
+
+		/// The origin that is allowed to create providers
+		type CreateProviderOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = Self::AccountId>;
+
+		/// The origin that is allowed to create providers via governance
+		type CreateProviderViaGovernanceOrigin: EnsureOrigin<Self::RuntimeOrigin>;
```

Comment:
These can be named the same. CreateProviderOrigin since they run on different runtimes.

Reply:
What do you suggest?

Reply:
We cannot because governance origins cannot use `AccountId`.

Reply:
Maybe we can pair.